site/_docs/encodings.md

```diff
@@ -123,6 +127,41 @@ DIRECT_V2     | PRESENT         | Yes      | Boolean RLE
               | DATA            | No       | Unbounded base 128 varints
               | SECONDARY       | No       | Unsigned Integer RLE v2
 
+In ORC 2.0, DECIMAL and DECIMAL_V2 encodings are introduced and scale
+stream is totally removed as all decimal values use the same scale.
+There are two difference cases: precision<=18 and precision>18.
+
+### Decimal Encoding for precision <= 18
+
+When precision is no greater than 18, decimal values can be fully
+represented by 64-bit signed integers which are stored in DATA stream
+and use signed integer RLE.
+
+Encoding      | Stream Kind     | Optional | Contents
+:------------ | :-------------- | :------- | :-------
+DECIMAL       | PRESENT         | Yes      | Boolean RLE
+              | DATA            | No       | Signed Integer RLE v1
+DECIMAL_V2    | PRESENT         | Yes      | Boolean RLE
+              | DATA            | No       | Signed Integer RLE v2
```

Comment:
Will there be a new RLE version in ORC 2.0? Since we already have 2 RLE versions this is confusing (<0.11 version uses RLE v1 and >=0.12 uses RLE v2). 
If this is newer RLE version then we should probably rename it to RLE v3.
If there will not be a new RLE version then we should probably not use the old RLE v1.

Reply:
For decimals, current Integer RLE will be used. As you have explained, I agree that we should not use old RLE v1.

Reply:
I think we should keep RLE v1 as an option. The C++ writer currently does not support RLE v2 (we are working on it). We don't want the new decimal writer to have dependency on that.

Reply:
@xndai Vertica is interested in getting RLE v2 for C++ as well. Do you think we can collaborate on getting this in quickly?

Reply:
@majetideepak We are already working on it and doing test & benchmark. Will contribute back  but may not be that soon.

Reply:
Some part of this discussion is about the new ORC format and existing reader compatibility is not a requirement, until we switch to the new format as a default.